api/middleware/responseLogger.go

```diff
@@ -66,15 +74,27 @@ func logRequestAndResponse(c *gin.Context, duration time.Duration, status int, r
 		}
 	}
 
+	title := rlm.computeLogTitle(status)
+
+	rlm.printRequestFunc(title, c.Request.RequestURI, duration, status, request, response)
+}
+
+func (rlm *responseLoggerMiddleware) computeLogTitle(status int) string {
 	logPrefix := prefixDurationTooLong
 	if status == http.StatusBadRequest {
```

Comment:
I still hate those else-if's :)

Reply:
I couldn't find a better approach in order to treat the 'duration too long' vs various http codes (from which 2 are treated separately)